recipes/elk/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "elk",
+  "name": "Elk",
+  "version": "0.1.0",
```

Comment:
please change the version of the recipe to `1.0.0`

Reply:
I changed it to 1.0.1 because the hook was complaining about changing the package without a version bump after I set it to 1.0.0